tls/vibe/stream/openssl.d

```diff
@@ -1041,7 +1032,7 @@ final class OpenSSLContext : TLSContext {
 			// as it is always enabled by default.
 			// https://github.com/openssl/openssl/issues/1437
 			// https://github.com/openssl/openssl/commit/2ecb9f2d18614fb7b7b42830a358b7163ed43221
-			static if (OPENSSL_VERSION_NUMBER >= 0x10200000 && OPENSSL_VERSION_NUMBER < OPENSSL_MAKE_VERSION(1, 1, 0, 0)) {
+			static if (OPENSSL_VERSION_AT_LEAST(1, 0, 2) && OPENSSL_VERSION_BEFORE(1, 1, 0)) {
```

Comment:
Looks like this introduce a functional change (?)

Reply:
Because the check was wrong:
```D
import std.format;
pragma(msg, format!"%X"(OPENSSL_MAKE_VERSION(1, 0, 1, 0)));
pragma(msg, format!"%X"(OPENSSL_MAKE_VERSION(1, 1, 0, 0)));
pragma(msg, format!"%X"(OPENSSL_MAKE_VERSION(3, 0, 0, 0)));
```

```shell
1000100F
1010000F
3000000F
```

Reply:
I'll revert this specific line for now